spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  fixtures :users
+  context 'create a simple user when have a valid input' do
+    it 'check if the input is valid' do
+      # Setup & Exercise
+      user = User.new(name: 'usuario')
+      # Verify
+      expect(user.name).to match(/[a-zA-Z]/)
+    end
+  end
+  context 'can not create a user without a name' do
+    it 'validate when the input is empty' do
+      # Setup
+      user = User.new
+      # Exercise
+      user.validate
+      # Verify
+      expect(user.errors[:name]).to include("can't be blank")
```

Comment:
RSpec has the `be_<predicate>` matcher, where predicate can any predicate method that exists in the object under test. Rails models have the `valid?` method, so you could've written this as:
```rb
it 'requires the name' do
  user = User.new(name: nil) # make the setup more explicit
  expect(user).not_to be_valid
  expect(user.errors[:name]).to include("can't be blank")
end
```

Reply:
Also adding this type of comments is unnecessay. Comments should explain why things are implemented as they are when the code is not clear enough (and it's not possible easy to make it more readable).